tests/commands/test_build.py

```diff
@@ -462,8 +474,31 @@ def test_build_multiple_with_charmcraft_yaml_managed_mode(
     )
 
 
+@pytest.mark.parametrize(
+    "charmcraft_yaml, metadata_yaml",
+    [
+        [
+            dedent(
+                """\
+                type: charm
+                """
```

Comment:
This is a bit misleading. Please either include the whole thing here as a string template or don't parametrize it